addon/components/frost-list.js

```diff
@@ -84,16 +86,16 @@ export default Component.extend({
 
       // State
       _rangeState: {
-        anchor: null,
-        endpoint: null
+        anchor: Ember.Object.create(),
+        endpoint: Ember.Object.create()
       }
     }
   },
 
   // == Computed Properties ===================================================
 
   @readOnly
-  @computed('expandedItems.[]', 'items.[]', 'selectedItems.[]', 'itemComparator')
+  @computed('expandedItems.[]', 'items.[]', 'selectedItems.[]', '_itemComparator')
   _items (expandedItems, items, selectedItems, itemComparator) {
```

Comment:
Probably makes sense for your variable to match the source - i.e. `_itemComparator`